sdk/ts-mcp/README.md

```
@@ -0,0 +1,342 @@
+- `createAppSession(definition: app.AppDefinitionV1, sessionData: string, quorumSigs: string[], ...)`
+- `submitAppState(appStateUpdate: app.AppStateUpdateV1, quorumSigs: string[]): Promise<void>`
+- `submitAppSessionDeposit(appStateUpdate: app.AppStateUpdateV1, quorumSigs: string[], ...)`
+- `getAppSessions(options?): Promise<{ sessions: AppSessionInfoV1[]; ... }>`
+- ...
+```
+
+### `get_rpc_method`
+
+Get the RPC wire format for a clearnode method. Matches `clearnode/docs/API.md`.
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)